backend/corpora/common/entities/collection.py

```diff
@@ -138,11 +138,11 @@ def reshape_for_api(self) -> dict:
             for link in result["links"]
         ]
 
-        result["datasets"] = [ds for ds in result["datasets"] if not ds.get("tombtone")]
+        result["datasets"] = [ds for ds in result["datasets"] if not ds.get("tombstone")]
```

Comment:
ah nice catch on typo! I had to read it twice to notice lol

Does your IDE provide spellchecking? I know PyCharm and VS Code both have that 😆